docker/Dockerfile

```diff
@@ -356,7 +355,8 @@ RUN apt-get update && \
 RUN pip install --no-cache-dir --upgrade \
         certifi${CERTIFI_VERSION} \
         ipython${IPYTHON_VERSION} \
-        urllib3${URLLIB3_VERSION}
+        urllib3${URLLIB3_VERSION} \
+        python-snappy
```

Comment:
why?

Reply:
python snappy doesn't install correctly without first apt install and then force pip install